### DISCUSSION TITLE
Losing place in books

### DISCUSSION THREAD

**Discussion by deadmonkies** [advplyr/audiobookshelf#605] [Q&A]
Hello, I've got audiobookshelf deployed via a docker container on my NAS that is automatically updated with watchtower. The container is accessible over the internet through a caddy reverse proxy, also running as a docker container. I exclusively use the android app (currently 0.9.45-beta) to listen to my books, and generally download the books to my phone whenever I start a new one. I've had frequent issues with losing my place in books after I stop/restart playback. The book always starts back at the very beginning, not to a random location in the book. I'm not sure what logs or diagnostics I can provide to help track this down, but I'm happy to give whatever info I can to fix this.

Thanks

**Comment by advplyr** [comment]
If you can break this down into specific reproducible steps that would help. Here is something you can try that would help,

1. Open mobile app and connect to server
2. Start book (not downloaded) and start listening
3. Open web app in desktop browser and go to config -> logs
4. Select Debug logs in the dropdown
5. While still streaming on the mobile app note if you see a logs every 6 seconds with "syncSession" and the total time listening.

**Reply by deadmonkies** [reply]
So I think I've figured out how to reproduce this issue. I do almost all my listening through the mobile app with downloaded books. I also do almost all my listening while in my car using bluetooth to the stereo. If I turn off the car without disconnecting bluetooth first, the next time I play the same book, it will (almost?) always start back at the beginning. If I manually disable bluetooth on my phone, then turn off the car, I do not lose my place.

Additionally, I tried what you suggested. When streaming (not downloaded) a book on my phone while sitting at my desk, I do not see sync session showing up on the web logs

![image](https://user-images.githubusercontent.com/6312857/172254279-fdb39ec3-5cc9-46eb-8545-ba341edd9f92.png)
